contracts/hardhat.config.ts

```diff
@@ -151,13 +151,15 @@ const config: HardhatUserConfig = {
   },
   docgen: {
     exclude: [
-      "token",
-      "test-contracts",
-      "proxies",
-      "tools",
-      "interfaces/tools",
-      "tokenBridge/mocks",
-      "tokenBridge/lib/StorageFiller39.sol",
+      "_testing",
+      "bridging/token/utils/StorageFiller39.sol",
+      "bridging/token/CustomBridgedToken.sol",
+      "governance/TimeLock.sol",
+      "libraries",
+      "messaging/libraries/MessageHashing.sol",
+      "security/access/PermissionsManager.sol",
+      "security/reentrancy/TransientStorageReentrancyGuardUpgradeable.sol",
```

Comment:
why are we excluding these?

Reply:
added libraries back in

The remaining specified *.sol files result in almost-empty doc pages